source/guide_gitea.rst

```diff
@@ -152,6 +152,9 @@ Finishing installation
 
 Point your browser to your gitea URL and finish the installation.
 
+.. note:: Gitea prevents you from logging in with private keys if you
+ don't use the built-in ssh server. Login via password still works as expected.
```

Comment:
Opening ports is not yet possible on uberspace 7, so the built-in server is not an option. I think we should nevertheless warn people about using the same key for SSH and git. Enabling both ssh and git using the same key would be awesome, but I don't think that's doable.

Reply:
Ah so that's the issue. I just followed the guide, added my ssh key (same as Uberspace) to Gitea and was locked out. I spent some time figuring out why putty wouldn't let me in. That's why I thought I'd update this notice.

So maybe something more along the lines of this?
```
Using the same public key for Gitea and your Uberspace will prevent you 
from logging in with this key via SSH. Login via password still works as expected. 
```

Reply:
Maybe include a hint that `authorized_keys` can be edited via the Dashboard if this happens?

Reply:
> Using the same public key for Gitea and your Uberspace will prevent you from logging in with this key via SSH. Login via password still works as expected, you can use the [Dashboard](https://uberspace.de/dashboard/authentication) to manage your keys.